src/components/NoticeOfWithdrawal/RecordToBeWithdrawn.vue

```diff
@@ -62,32 +64,132 @@
           id="withdrawal-filing-date"
           cols="12"
           sm="9"
+          class="grey-text"
         >
           {{ getFormattedFilingDate() }}
         </v-col>
       </v-row>
+      <v-divider class="mt-2 mx-4" />
```

Comment:
Don't be afraid to use whitespace between logical blocks (or around a divider) 😃

Reply:
Updated